docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Install and upgrade issues on Kubernetes
+headerTitle: Install and upgrade issues on Kubernetes
+linkTitle: Install and upgrade issues
+description: Troubleshoot issues encountered when installing or upgrading YugabyteDB Anywhere on Kubernetes.
+menu:
+  preview_yugabyte-platform:
+    identifier: install-upgrade-kubernetes-issues
+    parent: troubleshoot-yp
+    weight: 10
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li>
+    <a href="../vm/" class="nav-link">
+      <i class="fa-solid fa-building"></i>
+      VM
+    </a>
+  </li>
+
+  <li>
+    <a href="../kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+You might encounter issues during installation and upgrade of YugabyteDB Anywhere.
+
+If you experience difficulties while troubleshooting, contact [Yugabyte Support](https://support.yugabyte.com).
+
+## Unable to schedule a pod
+
+There can be multiple reasons behind the failure of pod scheduling for YBA pods. The following can be the reasons behind it. Follow to read about [Node selection in kube-scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/#kube-scheduler-implementation).
+
+### Do we have sufficient resources?
+
+**Symptom**
+
+YBA pods are in the Pending state.
+
+```sh
+## NAME                 READY   STATUS    RESTARTS   AGE
+## yw-test-yugaware-0   0/4     Pending   0          2m30s
+```
+
+**Debug**
+
+You can use the following commands to check the information about the pod. For more information, see [Kubernetes Troubleshooting guide](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-that-is-too-big-for-your-nodes).
+
+  ```sh
+  kubectl describe pod <POD_NAME> -n <NAMESPACE>
+
+  ## output
+  ## Events:
+  ##   Type     Reason             Age                From                Message
+  ##   ----     ------             ----               ----                -------
+  ##   Warning  FailedScheduling   56s                default-scheduler   0/2 nodes are available: 2 Insufficient cpu.
+  ```
+
+**Resolution**
+
+1. Please ensure you have enough resources in the kubernetes cluster to schedule the YBA pods. For more information, see [Prerequisites](https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/#hardware-requirements)
+
+2. Modify the YBA pods resources configuration. Follow the [YBA resources configuration](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#customize-yugabytedb-anywhere) to understand overrides.
+
+### Node selectors/Taints & Tolerations/Affinity
+
+**Symptom**
+
```

Comment:
We should mention here what user is supposed to see, instead of just the output of command. Also it would be better to specify the command which needs to be run to get the output shown here.

You will see...